server/views/pages/sessionSummary/_sessionsSummaryCountsBox.njk

```diff
@@ -0,0 +1,35 @@
+<section class="govuk-!-margin-bottom-8">
+
+  <h2 class="govuk-heading-m">Sessions summary</h2>
+
+  <div class="govuk-grid summary-counts-grid">
+    {# Count of sessions that are overdue #}
+    <div class="govuk-grid-item">
+      <p class="govuk-hint">Overdue inductions and reviews</p>
+      <div class="summary-count-container">
+        <span class="govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold" data-qa="overdue-sessions-count">10</span>
```

Comment:
Hardcoded counts for now (lines 10, 19 and 28)
Subsequent PRs will replace these with the counts we get from the API call